auditbeat/auditbeat.yml

```diff
@@ -0,0 +1,78 @@
+###################### Auditbeat Configuration Example #######################
+
+# This file is an example configuration file highlighting only the most common
+# options. The auditbeat.full.yml file from the same directory contains all the
+# supported options with more comments. You can use it as a reference.
+#
+# You can find the full configuration reference here:
+# https://www.elastic.co/guide/en/beats/auditbeat/index.html
+
+auditbeat.modules:
+- module: audit
+  metricsets: ["kernel"]
+  kernel.audit_rules:
+    # Define audit rules here.
+    # Create file watches (-w) or syscall audits (-a or -A). For example:
+    #-w /etc/passwd -p wa -k identity
+    #-a always,exit -F arch=b32 -S open,creat,truncate,ftruncate,openat,open_by_handle_at -F exit=-EPERM -k access
+
+#==========================  Modules configuration ============================
+auditbeat.modules:
```

Comment:
The organization of the config file looks weird to me. With the `audit` module above the `Modules configuraiton` line.